launch/launch/actions/unset_environment_variable.py

```diff
@@ -40,12 +40,13 @@ def __init__(
 
     @property
     def name(self) -> List[Substitution]:
-        """Getter for the name of the environment variable to be unset"""
+        """Getter for the name of the environment variable to be unset."""
         return self.__name
 
     def execute(self, context: LaunchContext) -> None:
         """Execute the action."""
-        name = perform_substitutions(context, self.name)
+        name = perform_substitutions(
+            context, self.name)
```

Comment:
Why was this needed?

Reply:
I thought that was the line that was too long, but turns out that it wasn't. Reverted.